web/src/custom_profile_fields_ui.ts

```diff
@@ -201,6 +202,39 @@ export function initialize_custom_user_type_fields(
     return user_pills;
 }
 
+export function initialize_profile_user_type_pills(user_id: number): void {
+    const field_types = realm.custom_profile_field_types;
+
+    for (const field of realm.custom_profile_fields) {
+        if (field.type !== field_types.USER.id) {
+            continue;
+        }
+        const raw = people.get_custom_profile_data(user_id, field.id)?.value;
```

Comment:
Would be good to rename this variable something like `raw_field_value` and then below it can be `value` can be replaced with `field_value`.